### PR TITLE
Fpclass

### PR DESCRIPTION
**Pull request by jtlap**
replace the previous pr which presented weird fails

### CODE REVIEW

File: test/doc/core/is_bit_equal.hpp [hunk 1]
```diff
@@ -0,0 +1,32 @@
+// revision 0
```
Comment (jfalcou) [inline test/doc/core/is_bit_equal.hpp:1]: useless file, remove

File: test/doc/math/pow.cpp [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
 #include <iostream>
+#include <cmath>
 
 int main()
 {
-  eve::wide pf = {2.0f, 3.0f, -4.0f, 2.0f, 2.0f,
+  eve::wide qf = {0.0f, 3.0f, -4.0f, 2.0f, 2.0f,
                   eve::inf(eve::as<float>()), eve::minf(eve::as<float>()), eve::nan(eve::as<float>())};
-  eve::wide qf = {4.0f, 1.0f, -1.0f,  0.5f,  0.0f, 2.0f, -3.0f,  2.5f};
+  eve::wide pf = {4.0f, 1.0f, -1.0f,  0.5f,  0.0f, 0.0f, -0.0f,  2.5f};
```
Comment (jfalcou) [inline test/doc/math/pow.cpp:10]: why changing those values ?

**Review by jfalcou — CHANGES_REQUESTED** [review]

File: test/doc/core/is_bit_equal.hpp (hunk 1 above)
Comment (jtlap) [reply test/doc/core/is_bit_equal.hpp:1]: done

File: test/doc/math/pow.cpp (hunk 2 above)
Comment (jtlap) [reply test/doc/math/pow.cpp:10]: I had an error due to is_minf and this made it visible

**Review by jfalcou — APPROVED** [review]

**Pull request merged**